ColorUtility.cpp

```diff
@@ -0,0 +1,49 @@
+#include "ColorUtility.h"
```

Comment:
Be careful of files whose name ends in `Utility`, `Manager`, `Handler`, `Helper`
It means we are not clear about the contents of the file. The file will end up attracting unrelated functionality.
In this case, the file already has multiple reasons to change - fixes in the ColorPair translation, as well as customer requests to add different formatting methods like HTML, csv, etc.

Reply:
Noted. Will endeavor to make the files names more relevant / specific